project_parameters.Config.yaml

```
@@ -115,6 +115,9 @@ gene_markers_file_name_annotation_module: "<filename>.tsv" # File name with the
 method_annotation_module: "all" # Options: "all" (for using singler and a list of gene markers), "singler" (for using only singler method for both broad and fine annotation)
 gene_name_convention_update_annotation_module: "NO" # Options: "YES" or "NO" (default). Set to "YES" if the gene names in the gene markers file do not align with the naming convention of genes in the reference genome.
 
+ct_palette_file_broad: "cell_types_palette.tsv"  # File name with the cell types. It needs to be in `tsv` format. Please ensure cell type names exist and match the ones in the `.figures/palettes/cell_types_palette.tsv`. Alternatively, add in that folder and use your own table.
```

I think the explanation for using your own palette file could be made a little more clear, e.g.

```suggestion
ct_palette_file_broad: "cell_types_palette.tsv"  # File name with the cell types. It needs to be in `tsv` format. Please ensure cell type names exist and match the ones in the `.figures/palettes/cell_types_palette.tsv`. Alternatively, create a tsv file with your own palette and add it to the `.figures/palettes/` folder. 
```

---

See here 2fe75a03a60ac0a86900d1ce04a2ffb92f8cfda6.